src/main/java/seedu/taskman/storage/XmlSerializableTaskMan.java

```diff
@@ -41,7 +45,8 @@ public XmlSerializableTaskMan() {}
     public XmlSerializableTaskMan(ReadOnlyTaskMan src) {
         //TODO: writing tasks and events
         //implemented XmlAdaptedTask(Activity activity) for now
-        tasks.addAll(src.getActivityList().stream().map(XmlAdaptedTask::new).collect(Collectors.toList()));
+        events.addAll(src.getActivityList().stream().filter(activity -> activity.getType().equals(Activity.ActivityType.EVENT)).map(XmlAdaptedEvent::new).collect(Collectors.toList()));
```

Comment:
Not sure if over 160 chars long, but it splitting to a new line might be a good idea.